### DISCUSSION TITLE
general

### DISCUSSION THREAD

**Discussion by PierreGuede** [dehboris/trombi-al-2021#10] [General]
Bonjour monsieur!  j'ai déjà fini l'excercice et j'ai déjà envoyé mais quand je faisais le git merge de ma branche upstream pour la branche main je tombe sur celui de charle! mais après plusieurs commit je suis ne trouve pas mon excerice dans le pull request mais pluot dans le menu code où se trouve votre commit et celui de charles! c'est normal?

**Comment by dehboris** [comment]
Oui c'est normal, il faut fusionner tes données et ceux Charles  en suivant les **consignes**

**Reply by PierreGuede** [reply]
Monsieur c'est normal que depuis que je tape la commande "git push origin main" il m'écrit:
remote: Permission to dehboris/trombi-al-2021.git denied to PierreGuede.
fatal: unable to access 'https://github.com/dehboris/trombi-al-2021/': The requested URL returned error: 403
